test/utils/testEnv.ts

```diff
@@ -48,6 +50,21 @@ const setStakingPool = async (
   return stakingPool;
 };
 
+const migrateSetStakingPool = async (
```

Comment:
I think `setMigrageStaking` would be the better choice for a unified name convention.